app/models/page.rb

```diff
@@ -15,15 +16,15 @@ class Page
   field :role  # only relevant for virtual pages
   field :active, type: Boolean, default: false
   field :layout, type: String, default: 'default'
-  field :meta_description, localize: Slices::Config.i18n?
-  field :title, localize: Slices::Config.i18n?
+  field :meta_description, localize: Slices::Config.i18n?, default: ''
+  field :title, localize: Slices::Config.i18n?, default: ''
```

Comment:
https://jira.mongodb.org/browse/MONGOID-4054. I think add that link as a comment somewhere here so reader knows why the defaults are being set. Then eventually it'll be fixed and released in Mongoid and this change can be reverted.